furusiya/model/entities/fire.py

```diff
@@ -0,0 +1,34 @@
+import colors
+from constants import DELTA_UP, DELTA_DOWN, DELTA_LEFT, DELTA_RIGHT
+from game import Game
+from model.entities.game_object import GameObject
+from model.config import config
+
+
+class Fire(GameObject):
+    def __init__(self, x, y):
+        super().__init__(x, y, '*', 'Fire', colors.red, blocks=False)
+
+        Game.events.bind('on_entity_move', self.on_entity_move)
+        Game.events.bind('on_turn_pass', self.on_turn_passed)
+
+        self.turns_passed_alight = 0
```

Comment:
I would rather make this `turns_left_alight`. That way, if anything happens, we can increment the amount.